Fixed global variables (#2174)
ibexa/documentation-developer: docs/cdp/cdp.md

## What is Ibexa CDP

[[= product_name_cdp =]] helps you solve one of the hardest challenges facing business world today:
building unique experiences for your customers.

> With [[= product_name_cdp =]] you will be able to track and aggregate data of your customers' activity on multiple channels.

Reviewer: ```suggestion
With [[= product_name_cdp =]] you are able to track and aggregate data of your customers' activity on multiple channels.
```
